module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3405,6 +3461,7 @@ protected function getItemsWithBibsForTransactions(
         }
         // Fetch items and collect bib id mappings if available:
         $itemIds = [];
+        $innReachIndices = [];
```

Comment:
This array gets populated but never seems to be used for anything. Is something missing, or can something be simplified/removed?